src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -187,7 +187,8 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     )
 
     val message = executionResult match {
-      case SuccessfulExecution(outputs, returnCode, hash) => WorkflowActor.CallCompleted(key, outputs, returnCode, hash)
+      case SuccessfulExecution(outputs, returnCode, hash) =>
+        WorkflowActor.CallCompleted(key, outputs, returnCode, if (workflowDescriptor.writeToCache) hash else "")
```

Comment:
Just out of curiosity, what do you mean they don't exist post compilation ? I thought case classes were actually bigger than normal classes because they provide additional methods (also all their constructor parameters are all attributes).